lib/src/kbasesearchengine/search/SortingRule.java

```diff
@@ -1,8 +1,15 @@
 package kbasesearchengine.search;
 
 public class SortingRule {
+
+    //object name is not sortable right now. Just going to use keyName
+//    public boolean isObjectName;
+
     public boolean isTimestamp;
-    public boolean isObjectName;
+    //cheat right not b/c workspaceId or access_group_id is not exposed to front end
+    public boolean isWorkspaceId;
```

Comment:
Can't quite parse the comment, but this should be isAccessGroupID

Reply:
okay